pkg/api/error.go

```diff
@@ -39,6 +39,12 @@ type CloudErrorBody struct {
 	// The target of the particular error. For example, the name of the property in error.
 	Target string `json:"target,omitempty"`
 
+	// The category for the error, being either a user or server error
+	Category string `json:"-"`
```

Comment:
Suggest defining a `CloudErrorCategory` string type for this, if it can only be one of the two constants defined below.

Reply:
Created CloudErrorCategory string type as suggested.